mathesar_ui/src/component-library/tabs/TabContainer.scss

```diff
@@ -11,7 +10,8 @@
     margin: 0;
     padding: 0;
     list-style: none;
-    border-bottom: 1px solid var(--slate-300);
+    border-bottom: 1px solid var(--slate-200);
+    gap: 2rem;
```

Comment:
There seems to be a number of usages of `gap`. Please notice Sean's comment: https://github.com/centerofci/mathesar/pull/2146#discussion_r1054700591. 

We try not to use `gap` because it's not supported in older browsers (particularly previous versions of Safari) which still has considerable market share.

Please remove them in this PR, or create a separate issue to keep track of this.